internal/util/util.go

```diff
@@ -408,6 +414,21 @@ func EscapePath(path string) string {
 	return strings.ReplaceAll(path, "/", "%")
 }
 
+// EscapePath is a helper to apply the URL or legacy approach
+func EscapePath(path string) string {
+	url := EscapePathUrl(path)
+	if _, err := os.Stat(url); err == nil {
+		return url
+	}
+
+	legacy := EscapePathLegacy(path)
+	if _, err := os.Stat(legacy); err == nil {
```

Comment:
Are you sure?

Because for me it finds the backups in the backup folder and this function is used like this at every place:
`backupfile := filepath.Join(b.BackupDir(), util.EscapePath(b.AbsPath))`

And the `BackupMsg` simply prints the backup filename (without preceding backup path) only due to space limitations.

Reply:
It finds backups with URL encoded names, but doesn't find backups with legacy names, right?

My point is that `EscapePath()` itself doesn't find a legacy-named backup, so it always returns the URL-encoded name, not the legacy-encoded one.

Reply:
Argh, damn!
Thanks for the hint. I fixed it locally.